Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -431,13 +432,18 @@ It can efficiently compress both junk data and encrypted Wii data.
 
     <string name="select_dir">Select This Directory</string>
 
+    <!-- Android Exclusive Settings-->
+    <string name="wii_overlay_disables_controllers">Disable Unselected Wii Overlay Controllers</string>
+    <string name="wii_overlay_disables_controllers_description">Disable overlay GameCube controller when using Wii Remote and vice versa. Inconvenient if switching between GameCube and Wii games. Unimplemented with custom game settings.</string>
```

Comment:
I think it would be better to keep that sentence and one of the TODOs mentions modifying this string so we shouldn't forget. Users wouldn't understand why the setting was hidden and a hidden setting doesn't necessarily mean a non-functional setting. I'd rather add a sentence to prevent confusion.